cyw43/src/control.rs

```diff
@@ -1,5 +1,6 @@
 use core::cmp::{max, min};
 use core::iter::zip;
+use core::time::Duration;
```

Comment:
please use `embassy_time::Duration` instead.

(`core::time::Duration` is a bit bloated for embedded usage: it has u64 seconds + u32 nanoseconds, and requires expensive multiply/divide by 10^9 when doing math with it)